src/generators/go/presets/JsonPreset.ts

```diff
@@ -0,0 +1,17 @@
+import { FieldType, GoPreset } from '../GoPreset';
+
+/**
+ * Preset which adds basic JSON Tags for fields
+ * 
+ * @implements {GoPreset}
+ */
+export const GO_JSON_PRESET: GoPreset = {
+  struct: {
+    field({fieldName, content, type }) {
+      if (type === FieldType.field) {
+        return `${content} \`json:"${fieldName},omitempty"\``;
```

Comment:
I have problem with that approach, because people should have possibility to add new tags to the existing one, so then they need to regex tags and retrieve existing one and merge with new... That preset does not work here and causes problems. Maybe we should have new one?

Reply:
That... is a great question and something I did not consider. I think that might make sense yes 🤔 

Unless we want to wait until we face the use-case that a user wants to do it of course?

Reply:
It is right to assume people will want to add more tags in some cases. Not only about JSON or other serialization mechanisms, but others like database-related, etc. So 👍 to what @magicmatatjahu says.

However, I would say that if it's hard to do at this PR, maybe it makes sense to create a feature request and work after? Just a suggestion.

Reply:
I just noticed this problem :) You can do as you see fit, however I know it @felixjung has a problem with the preset for description, so that can be "fixed" in this PR.